### PR TITLE
Polkadot version upgrade

### PR DESCRIPTION
**Pull request by bobo-k2**
**Pull Request Summary**

This PR brings upgrade to the latest polkadot js version (8.3.1). There was many breaking changes which are fixed. Because of that the whole portal features needs to be tested in detail.

Lot of issues I had were related to proxy issue described here https://github.com/polkadot-js/api/issues/4656#issuecomment-1072068847

**Check list**
- [ ] contains breaking changes
- [ ] adds new feature
- [x] modifies existing feature (bug fix or improvements)
- [ ] relies on other tasks
- [ ] documentation changes

**This pull request makes the following changes:**

**Fixes**
- polkadot version compatibility fixes

**Changes**
- upgrade to the latest Polkadot js
- removed dependency to astar.js

**To-dos**
> Implement Polkadot type generation

### CODE REVIEW

File: src/hooks/useClaimAll.ts [hunk 1]
```diff
@@ -9,7 +9,8 @@ import { getIndividualClaimTxs } from './helper/claim';
 import { useCurrentEra, useCustomSignature } from './index';
 
 export function useClaimAll() {
-  const batchTxs = ref<ExtrinsicPayload[]>([]);
+  let batchTxs: ExtrinsicPayload[] = [];
```
Comment (impelcrypto) [inline src/hooks/useClaimAll.ts:12]: Could you share with me why you are using `let` instead of `ref`?

**Review by impelcrypto — APPROVED** [review]
Amazing!

File: src/hooks/useClaimAll.ts (hunk 1 above)
Comment (bobo-k2) [reply src/hooks/useClaimAll.ts:12]: Sure. Ref is a proxy object and I stumbled upon following issue https://github.com/polkadot-js/api/issues/4656#issuecomment-1072068847

**Comment by impelcrypto** [comment]
@bobo-k2 I'm going to resolve the conflicts

**Comment by impelcrypto** [comment]
Hi @bobo-k2,
I'm facing very weird ESLint errors after upgrading Polkadot dependencies on my end. Could you fix the conflicting files in the branch you are currently working on? I think the conflicting code will be resolved after copying the code below.

https://github.com/AstarNetwork/astar-apps/pull/389/files

[src/boot/api.ts](https://github.com/AstarNetwork/astar-apps/blob/cde43cddebe30af1b6848b40bcda524145052dfa/src/boot/api.ts)
[src/components/dapp-staking/ClaimAll.vue](https://github.com/AstarNetwork/astar-apps/pull/389/files#diff-1f9cd5e02ca719310ea819823e8a39bf1a723536a713dd99bcba35b0a02703c9)
[src/components/dapp-staking/Dapp.vue ](https://github.com/AstarNetwork/astar-apps/pull/389/files#diff-1759f30420258c59f34869492bf2aa7e1e01851faf8e76abf29f9ff236d7214f)
[src/components/dapp-staking/StakePanel.vue](https://github.com/AstarNetwork/astar-apps/blob/cde43cddebe30af1b6848b40bcda524145052dfa/src/components/dapp-staking/modals/ClaimRewardModal.vue)
[src/hooks/useApr.ts](https://github.com/AstarNetwork/astar-apps/blob/cde43cddebe30af1b6848b40bcda524145052dfa/src/hooks/useApr.ts)
[src/hooks/useTvl.ts](https://github.com/AstarNetwork/astar-apps/blob/cde43cddebe30af1b6848b40bcda524145052dfa/src/hooks/useTvl.ts)
[src/store/dapp-staking/actions.ts ](https://github.com/AstarNetwork/astar-apps/blob/cde43cddebe30af1b6848b40bcda524145052dfa/src/store/dapp-staking/actions.ts)

image

image

**Pull request merged**